decidim/decidim
# Bump @tiptap packages from v2.0.3 to v2.1.12

#### :tophat: What? Why?
We previously locked Tiptap to 2.0.x because of [a bug introduced in version 2.1.0-rc.2](https://github.com/ueberdosis/tiptap/issues/4347).

This bug seems now fixed in the latest releases through ueberdosis/tiptap#4354, so updating to version 2.1.12 and above should work fine.

I'm still locking Tiptap to a minor version to avoid such issues in the future with Tiptap updates. I think a safe strategy would be to update the Tiptap versions manually and in a controlled way while releasing new Decidim patch releases.

#### :pushpin: Related Issues
- Related to
  * #11492
  * ueberdosis/tiptap#4354
  * ueberdosis/tiptap#4347

#### Testing
- See that CI is green (particularly the editor system specs)
- Regenerate the development app
- Rebuild the assets
- Go to a Tiptap editor in the admin panel
- Paste in a Youtube video link and see that it is automatically converted to a video embed element
- Paste in any other link and see that it is automatically converted to a link to that URL
